.github/workflows/release-nightly.yml

```diff
@@ -1,10 +1,9 @@
 name: Release nightly
 
 on:
+  schedule:
+    - cron: "0 0 * * *" # every day at midnight
```

Comment:
@julienrbrt shouldn't we use cache/diff logic to only create a release when the checked out code is different since the last time it was checked out? Same for Docker releases.

cc @mazzy89

Reply:
That's indeed an another improvement we can do 👍🏾
This is at least better than currently (runs on every push)

Reply:
That should do it 👍